resources/servo.css

```diff
@@ -60,3 +60,11 @@ details[open]::-servo-details-content {
   display: block;
 }
 
+/*
+ * Until servo supports svg properly, make sure to at least prevent svg
+ * children from being layed out and rendered like usual html.
+ * https://github.com/servo/servo/issues/10646
+ */
+svg > * {
+  display: none;
```

Comment:
What I mean is changing the selector to something like `svg|svg *` with the namespace declared as `@namespace svg url(http://www.w3.org/2000/svg);`.

I'm not sure if this will actually result in a different effect, though, given, that `<svg>` in HTML5 is always implicitly under the SVG namespace (right?)